src/components/FeaturedListing/Card.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import PropTypes from 'prop-types/prop-types';
+
+import colors from 'styles/colors';
+import {
+  CardBody,
+  CardContainer,
+  Flex,
+  Image,
+  CardFooter,
+  CardImageContainer,
+  Tag,
+  TagContainer,
+  PersonImg,
+  Text,
+  Span,
+  IconContainer,
+} from './featured-listing.styles';
+import { ReactComponent as BedroomIcon } from '../../assets/icons/bedroom.svg';
```

Comment:
we should use absolute imports here and everywhere 
```suggestion
import { ReactComponent as AirConditionerIcon } from 'assets/icons/air-conditioner.svg';
```